testdata/webhooks/merge_request.json

```diff
@@ -159,18 +155,22 @@
         "type": "ProjectLabel",
         "group_id": 41
       }]
-    },
-    "reviewers": {
-      "previous": [],
-      "current": [
-        {
-          "id": 1,
-          "name": "User1",
-          "username": "user1",
-          "avatar_url": "http://www.gravatar.com/avatar/e64c7d89f26bd1972efa854d13d7dd61?s=40&d=identicon",
-          "email": "[REDACTED]"
-        }
-      ]
     }
-  }
-}
+  },
+  "assignees": [
+    {
+      "id": 6,
+      "name": "User1",
+      "username": "user1",
+      "avatar_url": "http://www.gravatar.com/avatar/e64c7d89f26bd1972efa854d13d7dd61?s=40\u0026d=identicon"
+    }
+  ],
+  "reviewers": [
+    {
+      "id": 6,
+      "name": "User1",
+      "username": "user1",
+      "avatar_url": "http://www.gravatar.com/avatar/e64c7d89f26bd1972efa854d13d7dd61?s=40\u0026d=identicon"
+    }
+  ]
+}
```

Comment:
Please make sure you end files with a newline. And that doesn't mean adding another line, it means the last line show end with e newline character. You can config your editor to automatically do this for you.

Reply:
Sorry, I updated my editor's settings.